evaluation/evaluation.md

```diff
@@ -1764,3 +1768,328 @@ display_merged_filtered_results(counts, titles, total_results, combinations, lab
 </table>
 </div>
 
+
+## Addon counts
+
+We want to train an ensemble model using the individual recommenders that we already have. To optimize this ensemble model, we need some training data. We have information about what addons different users have installed, so it would make sense to check if our ensemble model would also recommend these addons to the respective users.
+
+However, there is one fundamental conflict here: To be able to make recommendations, the collaborative recommender already needs some information about which addons a user has installed. Thus, we can only use a subset of a user's installed addons for evaluation. These addons are masked and the collaborative recommender then only uses the unmasked addons. The open question is how large this subset of masked addons should be.
```

Comment:
I think the question is more broad than just the "size" of the subset. More generally, we need to make an informed decision on the inclusion/selection criteria of the validation set given the optimization paradigm we have designed.